### ISSUE TITLE
revert ipfs gateway url

### ISSUE THREAD

**Issue by jakeloo** [nftlabs/nftlabs-sdk-ts#57]
we need to revert ipfs url to cloudflare-ipfs.com, customer reporting metadata not loading. 
also tested it myself where the same cid worked on cloudflare-ipfs.com but timed out on ipfs.io

**Comment by atbe** [comment]
We can do this, but it's only a bandaid to what is a bigger issue: customers are using a public gateway that should not be used in a production environment.

**Comment by atbe** [comment]
Also, there's a simple way to change this *without the customer having to wait for us to publish a new package version and forcing them to upgrade to it*:

```js
const sdk = new ThirdwebSDK(RPC_URL, {
  ipfsGatewayUrl: "https://cloudflare-ipfs.com/ipfs/",
});
```

I don't think we should get into the habit of publishing an entire release just to change a default config. That forces customers to upgrade to new package versions when they might not be ready for it.

**Comment by atbe** [comment]
Merged in #59
